Cant build with `cross`

**What happened**:

When running `make build-image` (or more exactly `build-linux-binary`) I get

```sh
make: *** [Makefile:94: build-linux-binary] Error 127
```

This also happens when running the failing step without `make`:

```sh
docker run --rm \
  -w /workspace \
  -v /path/to/quilkin:/workspace \
  -v ~/.cargo/registry:/usr/local/cargo/registry \
  -v /var/run/docker.sock:/var/run/docker.sock \
   --entrypoint=cross quilkin-build build --target x86_64-unknown-linux-gnu ; echo $?
127
```

**What you expected to happen**:

Quilkin to build successfully with `cross`.

**How to reproduce it (as minimally and precisely as possible)**:

```sh
docker run --rm \
  -w /workspace \
  -v /path/to/quilkin:/workspace \
  -v ~/.cargo/registry:/usr/local/cargo/registry \
  -v /var/run/docker.sock:/var/run/docker.sock \
   --entrypoint=cross quilkin-build build --target x86_64-unknown-linux-gnu
```

Just changing the `entrypoint` to `cargo` makes it work.

**Anything else we need to know?**:

I don't get any output from the `Makefile` or when running Docker manually but if I attach to the image and try to run `cross` manually I get the following:

```sh
docker run --rm -it \
  -v /path/to/quilkin:/workspace \
  -v ~/.cargo/registry:/usr/local/cargo/registry \
  -v /var/run/docker.sock:/var/run/docker.sock \
  -w /workspace \
  --entrypoint=/bin/bash quilkin-build 
```

```sh
root@6f37b81e4dcc:/workspace#  echo $PATH
/usr/local/cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin

root@6f37b81e4dcc:/workspace# echo $CARGO_HOME
/usr/local/cargo

root@6f37b81e4dcc:/workspace# cross build --target x86_64-unknown-linux-gnu
info: syncing channel updates for '1.59.0-x86_64-unknown-linux-gnu'

  1.59.0-x86_64-unknown-linux-gnu unchanged - rustc 1.59.0 (9d1b2106e 2022-02-23)

info: checking for self-updates
info: downloading component 'rust-src'
info: installing component 'rust-src'
sh: 1: cargo: not found
```

**Environment**:
- Quilkin version: 0.4.0
- Execution environment (binary, container, etc): -
- Operating system: WSL Ubuntu 20.04 under Windows 11
- Custom filters? (Yes/No - if so, what do they do?): No
- Log(s): -
- Others: -

--- a/build/Makefile
+++ b/build/Makefile
@@ -30,7 +30,6 @@ package_version := $(shell grep version -m 1 $(project_path)/Cargo.toml | awk '{
 
 common_rust_args := -v $(project_path):/workspace -w /workspace \
 					-v $(CARGO_HOME)/registry:/usr/local/cargo/registry
-docker_in_docker := -v /var/run/docker.sock:/var/run/docker.sock
 
 #   _____                    _
 #  |_   _|_ _ _ __ __ _  ___| |_ ___
@@ -91,17 +90,17 @@ binary-archive: ensure-build-image build-all-binaries
 
 # Build release and debug binaries for x86_64-unknown-linux-gnu
 build-linux-binary: ensure-build-image
-	docker run --rm $(common_rust_args) $(docker_in_docker) \
- 		--entrypoint=cross $(BUILD_IMAGE_TAG) build --target x86_64-unknown-linux-gnu
-	docker run --rm $(common_rust_args) $(docker_in_docker) \
- 		--entrypoint=cross $(BUILD_IMAGE_TAG) build --target x86_64-unknown-linux-gnu --release
+	docker run --rm $(common_rust_args) \
+ 		--entrypoint=cargo $(BUILD_IMAGE_TAG) build --target x86_64-unknown-linux-gnu
+	docker run --rm $(common_rust_args) \
+ 		--entrypoint=cargo $(BUILD_IMAGE_TAG) build --target x86_64-unknown-linux-gnu --release
 
 # Build release and debug binaries for x86_64-pc-windows-gnu
 build-windows-binary: ensure-build-image
-	docker run --rm $(common_rust_args) $(docker_in_docker) \
- 		--entrypoint=cross $(BUILD_IMAGE_TAG) build --target x86_64-pc-windows-gnu
-	docker run --rm $(common_rust_args) $(docker_in_docker) \
- 		--entrypoint=cross $(BUILD_IMAGE_TAG) build --target x86_64-pc-windows-gnu --release
+	docker run --rm $(common_rust_args) \
+ 		--entrypoint=cargo $(BUILD_IMAGE_TAG) build --target x86_64-pc-windows-gnu
+	docker run --rm $(common_rust_args) \
+ 		--entrypoint=cargo $(BUILD_IMAGE_TAG) build --target x86_64-pc-windows-gnu --release
 
 # Build release and debug binaries for x86_64-apple-darwin
 build-macos-binary:

--- a/build/build-image/Dockerfile
+++ b/build/build-image/Dockerfile
@@ -26,7 +26,7 @@ ENV RUSTUP_HOME=/usr/local/rustup \
 # Install packages
 RUN set -eux && \
     apt-get update && \
-    apt-get install -y jq wget zip build-essential libssl-dev pkg-config python3-pip && \
+    apt-get install -y jq wget zip build-essential libssl-dev pkg-config python3-pip g++-mingw-w64-x86-64 && \
     pip3 install live-server
 
 # Install htmltest
@@ -41,10 +41,10 @@ RUN wget https://static.rust-lang.org/rustup/dist/x86_64-unknown-linux-gnu/rustu
     rm rustup-init && \
     chmod -R a+w $RUSTUP_HOME $CARGO_HOME && \
     rustup component add rustfmt clippy && \
-    cargo install cross cargo-watch mdbook && \
+    rustup target add x86_64-pc-windows-gnu && \
+    cargo install cargo-watch mdbook && \
     cargo install cargo-about && \
     cargo install --locked cargo-deny && \
     rustup --version && \
     cargo --version && \
-    rustc --version && \
-    cross --version
+    rustc --version

